src/odemis/dataio/csv.py

```diff
@@ -98,7 +96,7 @@ def export(filename, data):
                 unit = "nm"
 
             # attach distance as first row
-            line_length = data.shape[1] * data.metadata[model.MD_PIXEL_SIZE][1]
+            line_length = data.shape[0] * data.metadata[model.MD_PIXEL_SIZE][1]
```

Comment:
Double check:
Probably this change was made on purpose but is the shape of the multiplication correct?
Is **0** meant to be multiplied with **1**?

Reply:
Yes, that was the bug. Actually, even for standard images, that's typically the case, because numpy stores the dimensions in reverse order.